### ISSUE TITLE
0.8.3 panics on Windows

### ISSUE THREAD

**Issue by andrewbanchich** [nats-io/nats.rs#122]
Running on macOS, everything works, but on Windows we are getting a huge panic that starts with:

```
thread '<unnamed>' panicked at 'could not load system certs: (Some(RootCertStore { roots: [OwnedTrustAnchor { subject: [49, 30, 48, 28, 6,
```

and ends with:

```
Custom { kind: InvalidData, error: BadDER })4, 33, 98, 102, 67, 112, 214, 213, 192, 7, 225, 2, 3, 1, 0, 1], name_constraints: None }] }), Custom { kind: InvalidData, error: BadDER })4, 33, 98, 102, 67, 112, 214, 213, 192, 7, 225, 2, 3, 1, 0, 1], name_constraints: None }] }), Custom { kind: InvalidData, error: BadDER })', ', ', C:\Users\username\.cargo\registry\src\github.com-1ecc6299db9ec823\nats-0.8.3\src\asynk\connector.rsC:\Users\username\.cargo\registry\src\github.com-1ecc6299db9ec823\nats-0.8.3\src\asynk\connector.rsC:\Users\username\.cargo\registry\src\github.com-1ecc6299db9ec823\nats-0.8.3\src\asynk\connector.rs:::404040:::141414
```

Everything in between looks like binary data.

As a test, I set `nats = "=0.8.2"` in Cargo.toml and everything ran with no panics.

**Comment by spacejam** [comment]
Is the certificate file being loaded actually accessible by the process?

**Comment by andrewbanchich** [comment]
Pretty sure it is, since to get it working all we do is change to `=0.8.2` and it works.

**Comment by stjepang** [comment]
Between `nats` 0.8.2 and 0.8.3 we replaced `webpki-roots` with `rustls-native-certs`. It looks like system certificates on Windows are not loading (not sure if rustls is unable to load them or if they're broken).

I found a related reported issue (https://github.com/ctz/rustls-native-certs/issues/5) and looks like it was already addressed in early 2020. The workaround is to simply skip certificates that can't be loaded and move on.

PR #136 adds the workaround.

@andrewbanchich We have just released version 0.9.0 - do you want to update to the latest version of this crate or would you like this workaround to also be backported to 0.8.x?

**Comment by andrewbanchich** [comment]
@stjepang  I've just updated to 0.9.0 but noticed that `async_nats::Subscription` is no longer `Stream`, but instead has a `next` method on it. Just curious - what's the reasoning behind this?

**Comment by abdelmonem17** [comment]
thank you for your great effort in developing library but next function uses a blocking technique so it will limit the usage of the library or you have to workaround anyway will lead to less performance.

**Comment by stjepang** [comment]
This should now be fixed in version 0.9.1.

**Comment by andrewbanchich** [comment]
Confirmed this is fixed. Thank you @stjepang !
